open_duck_mini_v2/open_duck_mini_v2.xml

```diff
@@ -0,0 +1,454 @@
+<mujoco model="onshape">
```

Comment:
Can you do some minimal formatting on this onshape generated XML? We have some instructions [here](https://github.com/google-deepmind/mujoco_menagerie/blob/main/CONTRIBUTING.md#xml-style).